partcad/src/partcad/__init__.py

```
@@ -39,6 +41,7 @@
     "part",
     "part_factory",
     "part_factory_step",
+    "part_factory_brep",
```

Ruff (0.8.2)

`part_factory_brep` may be undefined, or defined from star imports

(F405)